### DISCUSSION TITLE
自定义函数 Agg Function my_count is not implemented

### DISCUSSION THREAD

**Discussion by zbjbaggio** [apache/doris#12419] [A - General / Q&A]
show full functions in demo like 'my_count';是能查到的，但是执行select my_count(cost) from expamle_tbl;就报 Agg Function my_count is not implemented，请问是什么问题

**Comment by dataalive** [comment]
Doris version ?

**Reply by zbjbaggio** [reply]
1.1.1

**Comment by catpineapple** [comment]
show ' show full functions in demo like 'my_count'  '  and  the original error message of 'Agg Function my_count is not implemented' .
看看你的 ' show full functions in demo like 'my_count'  ' 命令返回信息和 'Agg Function my_count is not implemented'  的返回原始日志。

**Reply by zbjbaggio** [reply]
' show full functions in demo like 'my_count' ' 命令返回信息;
![image](https://user-images.githubusercontent.com/8317553/189789356-94758dfa-ebce-48ca-ae24-30f6a968c29f.png)
<HTML>
<BODY>
<TABLE><TR><TD>{&quot;object_file&quot;:&quot;org.apache.doris.common.util.URI@23d1698e&quot;,&quot;finalize_fn&quot;:&quot;_ZN9doris_udf13CountFinalizeEPNS_15FunctionContextERKNS_9BigIntValE&quot;,&quot;init_fn&quot;:&quot;_ZN9doris_udf9CountInitEPNS_15FunctionContextEPNS_9BigIntValE&quot;,&quot;merge_fn&quot;:&quot;_ZN9doris_udf10CountMergeEPNS_15FunctionContextERKNS_9BigIntValEPS2_&quot;,&quot;md5&quot;:&quot;055f93ffeab91c9688955e66175e34aa&quot;,&quot;update_fn&quot;:&quot;_ZN9doris_udf11CountUpdateEPNS_15FunctionContextERKNS_6IntValEPNS_9BigIntValE&quot;}</TD></TR></TABLE>

</BODY>
</HTML>
 'Agg Function my_count is not implemented' 的返回原始日志：
2022-09-13 10:10:57,499 WARN (doris-mysql-nio-pool-2|192) [StmtExecutor.execute():460] execute Exception. stmt[47, cd1b15f1485d4de4-8b2bb63042db73c7], errCode = 2, detailMessage = Agg Function my_count is not implemented

感谢

**Reply by catpineapple** [reply]
17644016496 is my wechat

**Comment by catpineapple** [comment]
The implementation of this UDF does not use vectorization ，Therefore the vectorized execution engine should be disabled。

**Comment by bin41215** [comment]
set enable_vectorized_engine = false;
